rulebooks/dt_webhook_event_example_rule.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Listen for events on dt_webhook
+  hosts: all
+  sources:
+    - dynatrace.event_driven_ansible.dt_webhook: #use dt_webhook as source for tests with local ansible-rulebook 
```

Comment:
Using "dynatrace.event_driven_ansible" as a prefix requires that the plugin is published to the Galaxy. Correct?

Reply:
Yes exactly.